### PR TITLE
gh actions: CI check improvement

### PR DESCRIPTION
**Pull request by PaulYuuu**
1. Rename requirements-travis.txt to requirements-ci.txt since we switch
to GitHub actions now.
2. Upgrade some pip packages to sync style/lint check update.

ID: 2135571
Signed-off-by: Yihuang Yu <yihyu@redhat.com>

### CODE REVIEW

**Comment by PaulYuuu** [comment]
Hello @luckyh @yanan-fu @YongxueHong @zhencliu, could you help review? Thanks in advance.

File: requirements-ci.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pylint
+inspektor==0.5.3
```
Comment (PaulYuuu) [inline requirements-ci.txt:2]: Actually, I also want to remove the version restriction of `inspektor`, then CI check will become real-time synchronization with upstream.

**Comment by PaulYuuu** [comment]
cc @vivianQizhu

**Review by YongxueHong — APPROVED** [review]

**Review by zhencliu — APPROVED** [review]

**Comment by PaulYuuu** [comment]
Thanks all. Merge the patch to allow CI checks to continue correctly.

**Pull request merged**